public/app/features/serviceaccounts/components/CreateTokenModal.tsx

```diff
@@ -109,7 +109,7 @@ export const CreateTokenModal = ({ isOpen, token, serviceAccountLogin, onCreateT
               }}
             />
           </Field>
-          {!isWithExpirationDate && (
+          {
```

Comment:
nit: I think you can remove the curly braces here and on line 121.

Reply:
Fixed